core/src/test/scala/unit/kafka/admin/ListConsumerGroupTest.scala

```diff
@@ -95,9 +95,18 @@ class ListConsumerGroupTest extends ConsumerGroupCommandTest {
     result = ConsumerGroupCommand.consumerGroupStatesFromString("Dead,CompletingRebalance,")
     assertEquals(Set(ConsumerGroupState.DEAD, ConsumerGroupState.COMPLETING_REBALANCE), result)
 
```

Comment:
Could we also add tests related to the new states in the other tests?

Reply:
Once the group type PR's are merged, the test params for new and old GC will be present so we can verify if everything is working as expected!

Reply:
done